semantic/src/site/globals/site.overrides

```diff
@@ -11,8 +15,112 @@ p strong {
   font-weight: @semiboldWeight;
 }
 
-.ui.basebody.container {
-  background: rgb(248, 248, 248);
+#main-container {
+  padding: 0;
+  position: relative;
+  min-height: 100vh;
+  // Max width is 1027px, from Semantic-UI Container
+}
+
+#user-area {
+  font-family: @userAreaFont;
+  color: @userAreaText;
+  display: flex;
+  position: fixed; // TO-DO: No fixed on mobile
+  background-color: @headersInteractiveLow;
+  padding-top: 10px;
+  padding-bottom: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
+  height: @headerHeight; // TO-DO: Adjust for mobile
+  justify-content: space-between;
+  //   top: 0,
+  //   zIndex: 1000,
+}
+
+#user-area.ui.container {
+  margin-left: 0px !important;
+  margin-right: 0px !important;
+}
+
+#user-area-left {
+  display: flex;
+  flex-direction: column;
+}
+
+#menu-bar {
+  // border: 1px white solid;
+}
+// Menu bar links
+#user-area a {
+  color: @userAreaText;
+  font-size: 14px;
+}
+
+// Not currently used, but should be appplied to the Link in the menu
+// bar that reflects the current page
+.selected-link {
+  font-weight: bold;
+}
+
+#org-selector {
+  font-size: 27px;
+  letter-spacing: 3.4px;
+  text-transform: uppercase;
+  // border: 1px white solid;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  max-height: 65px;
+  margin-top: 10px;
+}
+
+#org-selector .image {
+  margin-right: 20px;
+  height: 100%;
+}
+
+#user-area-user-menu .button {
+  margin-top: 10px;
+  border-radius: 20px;
+  background-color: @surfacesWhite;
+  color: @darkGrey;
+  min-width: 120px;
+}
+
+#content-area {
+  padding-top: @headerHeight + 25px;
+  padding-right: 25px;
+  padding-left: 25px;
+  padding-bottom: @footerHeight + 25px; // To allow for footer
+}
```

Comment:
I have also changed that to fit better in this [PR#593](https://github.com/openmsupply/application-manager-web-app/pull/619) not sure why it wasn't fitting correctly, and doesn't need to be changed now. I'm just flagging this, in case you have any feedback of why it shouldn't be changed.